### ISSUE TITLE
Finding only exact matches

### ISSUE THREAD

**Issue by gpeipman** [SPCSR/HelperFunctions#5]
Hi!

Is it somehow possible to control through script the results returned? Currently search returns all matches. If synonym is John then also Johnston is found. Any way to control it?

**Comment by estruyf** [comment]
@gpeipman if you want, you can add wildcard values to your synonym list. Like: john*

**Comment by gpeipman** [comment]
No-no. The problem is that I get currently also Johnston with John but I want there only to be John :)

**Comment by estruyf** [comment]
@gpeipman Are you doing your searches against the SharePoint results source or people results source?

@wobba any other option than disabling stemming when querying against the SharePoint results source?

**Comment by gpeipman** [comment]
SharePoint results source, not people one. And I'm using the script on Office 365.

**Comment by wobba** [comment]
@gpeipman and if you remove the script from the page, then you don't get Johnston? Could you grab the request and response from the query using your browser dev tools? To see what the query looks like and how it might have been rewritten bynthe query component.

**Comment by estruyf** [comment]
@gpeipman any update on this?
